docs/IR.md

```diff
@@ -1,127 +1,382 @@
-Open Neural Network Exchange (ONNX)
+Open Neural Network Exchange - ONNX
 =========
 
-ONNX is an open specification that consists of the following components:
+__Purpose__
+
+This document contains the normative specification of the semantics of ONNX. The .proto and .proto3 files found under the ‘onnx’ folder form the normative specification of its syntax. Commentary found in the .proto and .proto3 files are intended to improve readability of those files, but are not normative if they conflict with this document. Such conflicts should be reported as documentation bugs.
 
-1)  A definition of an extensible computation graph model
+__Notes on model validation__
 
-2)  Definitions of built-in operators and standard data types
+A [Python-based tool](../onnx/checker.py) is available to perform general validation of models against this specification.
```

Comment:
checker.py is a python wrapper, actually the real implementation is in C++.

Reply:
OK, will explain.